src/main/java/HttpBrowser.java

```diff
@@ -154,9 +154,17 @@ public boolean bodyHasDirectoryContents(String directoryPath) throws IOException
     }
 
     public String location() {
-        Header locationHeader = response.getFirstHeader("location");
+        return getHeaderValueFor("location");
```

Comment:
Should `"location"` be updated to `HttpHeaders.LOCATION`?

Reply:
Yes?